code/game/objects/items/weapons/storage/pouch.dm

```diff
@@ -0,0 +1,27 @@
+/obj/item/storage/pouch
+	name = "storage pouch"
+	desc = "A small storage pouch."
+	icon = 'icons/obj/contained_items/storage/pouch.dmi'
+	icon_state = "pouch"
+	contained_sprite = TRUE
+	use_sound = /decl/sound_category/rustle_sound
+	drop_sound = 'sound/items/drop/hat.ogg'
+	pickup_sound = 'sound/items/pickup/hat.ogg'
+	w_class = ITEMSIZE_SMALL
+	max_w_class = ITEMSIZE_SMALL
+	max_storage_space = 14 // 7 small items
+	storage_slots = 7
+	allow_same_size_storage = TRUE
```

Comment:
So my concern here is that this allows you to store a pouch within another similar pouch. While not a problem at this very moment with how the subtypes are defined, it should probably be proofed for the future whenever someone inevitable uses the base pouches.

Reply:
agreed